src/test/kotlin/com/github/julianhille/sqldelight/jscipherdriver/JsQueryTest.kt

```diff
@@ -0,0 +1,189 @@
+package com.github.julianhille.sqldelight.drivers.jscipher
+
+import com.squareup.sqldelight.Query
+import com.squareup.sqldelight.db.SqlCursor
+import com.squareup.sqldelight.db.SqlDriver
+import com.squareup.sqldelight.internal.Atomic
+import com.squareup.sqldelight.internal.copyOnWriteList
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+import kotlin.test.assertNull
+import kotlin.test.assertTrue
+
+class JsQueryTest {
```

Comment:
Just have a cohesive naming pattern as in`JsDriverTest.kt`, please and rename all functions.